blenderproc/python/utility/InstallUtility.py

```diff
@@ -183,11 +183,14 @@ def __call__(self, block_num, block_size, total_size):
 
             # Try to get major version of given blender installation
             major_version = None
-            for sub_dir in os.listdir(blender_path):
+            for root, sub_dirs, files in os.walk(blender_path):
                 # Search for the subdirectory which has the major version as its name
-                if os.path.isdir(os.path.join(blender_path, sub_dir)) and sub_dir.replace(".", "").isdigit():
-                    major_version = sub_dir
-                    break
+                # Note: For Blender (2.93.5 2021-10-06 arm64) on MacOS the dir is "/Applications/Blender.app/Contents/Resources/2.93/"
+                # That's why we need to search all the subdirectories recursively to find major version
+                for sub_dir in sub_dirs:
+                    if sub_dir.replace(".", "").isdigit():
+                        major_version = sub_dir
+                        break
```

Comment:
Hey,

I will check if that is true for other platforms as well and what blender will do in `3.x`. 

Then I would propose that we search for all subfolders: 

```python
all_contents_in_blender_folder = glob.glob(os.path.join(blender_path, "**"), recursive=True)
all_sub_folders = [os.path.basename(path) for path in all_contents_in_blender_folder if os.path.isdir(path)]
for sub_dir in all_sub_fodlers:
    if sub_dir.replace(".", "").isdigit():
        major_version = sub_dir
        break
```

Thanks for the awareness!